README.md

```diff
@@ -52,3 +52,38 @@ when accessing logs:
 docker-compose run remote logs --function myFunction --tail
 ```
 
+## Troubleshooting
+
+### Deploying as an AWS User with MFA
+
+If deploying as an AWS user with 2FA, you will need to generate a session token to go along with temporary credentials with an expiration time. After configuring the AWS CLI on your local machine, run the following commands (after filling in the ``aws-account-id`` (173432075717 for ``bhdevacct``), ``aws-username`` and ``token-from-MFA-device`` fields) to get temporary credentials and set them as environment variables:
```

Comment:
I believe the AWS account ID is low-risk, but not sure if we want to keep it in Git. I can remove it if needed.